jerry-core/jerry-api.h

```diff
@@ -139,122 +139,86 @@ typedef bool (*jerry_external_handler_t) (const jerry_api_object_t *function_obj
 typedef void (*jerry_object_free_callback_t) (const uintptr_t native_p);
```

Comment:
I don't think that is should be removed, as it is definition of function type without implementation (and so, without comments) in the engine. Instead, comments, describing the arguments, should be added to the definition.